oc_cdtapi/BitbucketAPI/models/errors_errors_inner.py

```
@@ -0,0 +1,91 @@
+import re  # noqa: F401
+import json
+
+from pydantic import BaseModel, ConfigDict, Field, StrictStr
+from typing import Any, ClassVar, Dict, List, Optional
+from typing import Optional, Set
+from typing_extensions import Self
+
+class ErrorsErrorsInner(BaseModel):
+    """
+    ErrorsErrorsInner
+    """ # noqa: E501
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)